tests/functional/testOnboarding.js

```diff
@@ -277,4 +279,65 @@ describe('Onboarding', function() {
     assert.equal(await vpn.getQueryProperty(queries.screenOnboarding.ONBOARDING_VIEW, 'currentIndex'), 3);
     assert.equal(await vpn.getSetting('onboardingStep'), 3);
   });
+
+  it.only('Verify data collection checkbox and setting are decoupled', async () => {
+
+    //This function completes onboarding, and checks that the gleanEnabled setting remains enabled throughout the flow
```

Comment:
For the sake of consistency, could you move the test description right above the test (so right above `it(...)`)?


also THANK YOU for adding tests, love to see it :)